client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -14,9 +15,12 @@ const PluginDetailsSidebar = ( {
 		demo_url = null,
 		documentation_url = null,
 	},
+	billingPeriod,
```

Comment:
Instead of passing the current `billingPeriod` as a prop, we have a state selector we can use

https://github.com/Automattic/wp-calypso/blob/0753f8792dc14abc16d52b99781b7ddb15bd9205/client/my-sites/plugins/plugins-browser/index.jsx#L138